.pre-commit-config.yaml

```diff
@@ -25,18 +25,26 @@ repos:
             name: flake8-cython
             args: ["--config=python/.flake8.cython"]
             types: [cython]
-    - repo: https://github.com/cheshirekow/cmake-format-precommit
-      rev: v0.6.11
+    - repo: local
       hooks:
-          - id: cmake-format
-            name: cmake-format
-            args: ["--config-files", "cmake/config.json", "--in-place", "--"]
-            types: [file]  # override `types: [cmake]`
-            files: \.(cmake(\.in)?)$|CMakeLists\.txt
-          - id: cmake-lint
-            args: ["--config-files", "cmake/config.json", "--"]
-            types: [file]  # override `types: [cmake]`
-            files: \.(cmake(\.in)?)$|CMakeLists\.txt
+            - id: cmake-format
+              name: cmake-format
+              entry: bash scripts/run-cmake-format.sh cmake-format
```

Comment:
Can't you just make the script executable and call it directly?

Reply:
Oh the script is already executable, so just `scripts/run-cmake-format` works fine too. I just wrote this command out of personal preference. I can change it if you like.

Reply:
Since you already have the shebang, I think it makes more sense to invoke the script directly.

Reply:
Done.